### PR TITLE
Openssh private keys markup

### PR DESCRIPTION
**Pull request by babenek**
OPENSSH keys are keys too

### CODE REVIEW

File: benchmark/scanner/scanner.py [hunk 1]
```diff
@@ -137,6 +137,7 @@ def check_line_from_meta(self, file_path: str, line_num: int) -> Tuple[LineStatu
                         self._increase_result_dict_cnt(row["Category"], "false_cnt")
                         return LineStatus.FALSE, project_id, file_id
         self.lost_cnt += 1
+        print(f"LOST: {file_path}:{line_num}", flush=True)
```
Comment (csh519) [inline benchmark/scanner/scanner.py:140]: It seems for debugging?
Would you like to add this line to PR?

Comment (babenek) [reply benchmark/scanner/scanner.py:140]: Yes. I think, the output should be added to point for missed markup.

Comment (csh519) [reply benchmark/scanner/scanner.py:140]: Okay let's leave this line.

**Review by csh519 — APPROVED** [review]
Thank you for markup.

LGTM :+1:

**Pull request merged**